solidity/hardhat.config.ts

```diff
@@ -173,17 +178,20 @@ const config: HardhatUserConfig = {
     },
     keepTechnicalWalletTeam: {
       default: 4,
+      goerli: 0,
       mainnet: "0xB3726E69Da808A689F2607939a2D9E958724FC2A",
     },
     keepCommunityMultiSig: {
       default: 5,
+      goerli: 0,
       mainnet: "0x19FcB32347ff4656E4E6746b4584192D185d640d",
     },
     treasury: {
       default: 6,
```

Comment:
Don't we need `goerli: 0,` here as well? When I'm running the deploy on goerli, I'm getting:
```
Error: ERROR processing /home/runner/work/tbtc-v2/tbtc-v2/solidity/deploy/05_deploy_bridge.ts:
Error: invalid address (argument="address", value=undefined, code=INVALID_ARGUMENT, version=address/5.5.0) (argument="_treasury", value=undefined, code=INVALID_ARGUMENT, version=abi/5.5.0)
```